sectionproperties/analysis/cross_section.py

```diff
@@ -190,11 +179,14 @@ def init():
             self.mesh_elements = elements
             self.mesh_attributes = attributes
 
+            # log.log(level=logging.DEBUG, msg=f"Section:\n")
+            # log.log(level=logging.DEBUG, msg=f"Tri6 Elements: {self.elements}")
+
             # initialise class storing section properties
             self.section_props = SectionProperties()
 
         if time_info:
```

Comment:
@connorferster, I think this should this be `self.time_info`, not `time_info`?

Reply:
Also on line 182 above this comment, is this commented logging code required

Reply:
@Agent6-6-6 Good idea! If we change these args to default to self.time_info then it could streamline things nicely.

Reply:
Done!